client/src/components/App.jsx

```diff
@@ -0,0 +1,63 @@
+import React, {useState, useEffect} from 'react';
+import styled from 'styled-components';
+import MainPage from './MainPage';
+import Login from './Login'
+import axios from 'axios';
+
+const App = () => {
+
+  const [isLoggedIn, setIsLoggedIn] = useState(false);
+  const [loginUserName, setLoginUserName] = useState('');
+
+  useEffect(() => {
+    axios.get('/api/login').then(res => setIsLoggedIn(res.data))
+  }, [])
+
+  const logout = () => {
+    axios.delete('/api/logout').then(res => setIsLoggedIn(res.data)).then(console.log('isLoggedIn:', isLoggedIn))
+  }
+
+  if (isLoggedIn === false) {
```

Comment:
In general, you can just check the value of the boolean rather than comparing it. E.g.

```
if (isLoggedIn) {
   // ...
} else {
   // ...
}
```